frontend/www/js/omegaup/arena/contest_practice.ts

```diff
@@ -21,43 +27,171 @@ OmegaUp.on('ready', () => {
     data: () => ({
       problemInfo: null as types.ProblemInfo | null,
       problem: null as ActiveProblem | null,
+      problems: payload.problems as types.NavbarProblemsetProblem[],
       clarifications: payload.clarifications,
     }),
+    methods: {
+      getMaxScore: (
+        runs: types.Run[],
+        alias: string,
+        previousScore: number,
+      ): number => {
+        let maxScore = previousScore;
+        for (const run of runs) {
+          if (alias != run.alias) {
+            continue;
+          }
+          const score = run.contest_score || 0;
+          if (score > maxScore) {
+            maxScore = score;
+          }
```

Comment:
nit:

```suggestion
          maxScore = Math.max(maxScore, run.contest_score || 0);
```

eso es un poquitín más corto. si no, de preferencia hay que expresar las cosas de tal manera que las variables vayan en el mismo orden para que sea más fácil de entender:

```typescript
          if (maxScore < score) {
            maxScore = score;
          }
```

Reply:
Listo,